packages/rrweb-snapshot/src/utils.ts

```diff
@@ -293,4 +264,103 @@ export function isElementSrcBlocked(tagName: string): boolean {
   );
 }
 
+const EMAIL_REGEX = new RegExp(
+  "[a-zA-Z0-9.!#$%&'*+=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:.[a-zA-Z0-9-]+)*",
+);
+const LONG_NUMBER_REGEX = new RegExp('d{9,16}'); // unformatted ssn, phone numbers, or credit card numbers
+const SSN_REGEX = new RegExp('d{3}-?d{2}-?d{4}');
```

Comment:
shouldn't this be `\d` to use the digit class?

<img width="545" alt="Screenshot 2023-10-13 at 12 12 29 PM" src="https://github.com/highlight/rrweb/assets/1351531/dab3b673-e799-4aee-bbe1-7131faa724f7">